src/basop/control.h

```diff
@@ -85,10 +98,10 @@ static __inline void incrFor (void) {
  *
  *****************************************************************************/
 #ifndef WMOPS
-#define WHILE( a) while( a)
+#define WHILE(a) while (a)
```

Comment:
Really?

Reply:
Yeah... There are a few instances where white spaces were adjusted. I did not push files that had only changes in white spaces, but I left the changes in files for which contents were adjusted too. I assumed we will run a script to normalize the code later?